drivers/backup/auth.go

```diff
@@ -566,6 +567,7 @@ func DeleteRoleFromUser(userName string, role PxBackupRole, description string)
 	fn := "DeleteRoleFromUser"
 	// First fetch the user ID of the user
 	clientID, err := FetchIDOfUser(userName)
+	log.Infof("Response from FetchIDOfUser %s", string(clientID))
```

Comment:
This function `func FetchIDOfUser(userName string) (string, error)` already return `clidID` as a `string`, don't need to `string(clientID)`

Reply:
resolved